backend/package.json

```diff
@@ -5,9 +5,8 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "env NODE_ENV=production node src/index.js",
-    "start:with-indexer": "env NEAR_IS_INDEXER_BACKEND_ENABLED=true npm run start",
```

Comment:
In current code this variable turns off publishing `wamp` subscriptions and aggregating stats data, and I don't see in which environment it is justified at the moment.

Reply:
Once we migrate to SingleStore we might collect data differently and thus we will need to switch off the current indexer implementation in favor of SingleStore indexer implementation. It is not there at the moment, and since you have already spent time removing it, we can merge it now and re-introduce it later when we will have more clarity on the topic, but would not you spend your time, we could easily keep it as is.

Reply:
I'm totally in to re-introduce it when we have another data source but Indexer.
Thanks!